.github/workflows/terraformPlan.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Terraform fmt
         id: fmt
         run: terraform fmt -check -recursive -diff=true
-        continue-on-error: true
+        if: always()
```

Comment:
This change is necessary to get us red- ❌ s when a step fails in our GH actions flow, `continue-on-error` ignores the error and drops a ✅